jv880.jucer

```diff
@@ -147,8 +153,8 @@
     </LINUX_MAKE>
     <VS2022 targetFolder="Builds/VisualStudio2022">
       <CONFIGURATIONS>
-        <CONFIGURATION isDebug="1" name="Debug" targetName="jv880"/>
-        <CONFIGURATION isDebug="0" name="Release" targetName="jv880"/>
+        <CONFIGURATION isDebug="1" name="Debug" targetName="jv880" headerPath="C:\Program Files (x86)\Steinberg\ASIOSDK\common\"/>

```

Comment:
Hmm will this work in CI?

Reply:
It should, since I left the ASIO flag disabled. It's just an extra search path for headers.